src/DataTable/index.tsx

```diff
@@ -138,30 +144,36 @@ export const DataTable = <T extends Record<string, unknown>>(
     setInitialRender(false);
   }, [data, editing]);
 
+  const TableRowRender = tableRow ? tableRow : TableRow
+
   return (
     <>
-    <TableThemeProvider>
-      <TableToolbar
-        canAdd={editing === null}
-        canDelete={selectedFlatRows.length > 0}
-        canEdit={selectedFlatRows.length === 1}
-        canReset={data.length !== initialData.length}
-        handleAdd={handleAdd}
-        handleDelete={handleDelete}
-        handleEdit={handleEdit}
-        handleReset={handleReset}
-      />
-        <StyledDataTable>
-          {/* The following divs are styled in DataTable/styled.tsx  */}
+      <TableThemeProvider>
+        {disableToolbar ? (
+          <TableToolbar
+          canAdd={editing === null}
+          canDelete={selectedFlatRows.length > 0}
+          canEdit={selectedFlatRows.length === 1}
+          canReset={data.length !== initialData.length}
```

Comment:
I was thinking about if any row is updated and the total of the rows is the same, this won't work because the evaluation is just with the length.

Reply:
Yep, that's a good catch. That's a bit of tech debt that we need to address and I've opened a ticket for it here: https://github.com/CrossroadsCX/datatable/issues/21